core/commands/filestore.go

```diff
@@ -150,7 +151,7 @@ For ERROR entries the error will also be printed to stderr.
 	Encoders: cmds.EncoderMap{
 		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, out *filestore.ListRes) error {
 			if out.Status == filestore.StatusOtherError {
-				return fmt.Errorf(out.ErrorMsg)
+				fmt.Fprintf(os.Stderr, "%s\n", out.ErrorMsg)
```

Comment:
This _might_ be okay.  Although i think we need to use PostRun here.  I created an issue for it n `go-ipfs-cmds`: https://github.com/ipfs/go-ipfs-cmds/issues/133.

Blocking for now to until others weigh in.

Reply:
@Stebalien confirmed in ipfs/go-ipfs-cmds#133 that we should be using PostRun here.